test/elli_tests.erl

```diff
@@ -361,6 +377,23 @@ way_too_big_body() ->
     ?assertMatch({error, closed},
                  hackney:post("http://localhost:3001/foo", [], Body)).
 
+too_big_request_line() ->
+    Path = binary:copy(<<"a">>, 1024 * 1001),
+    {ok, Socket} = gen_tcp:connect("127.0.0.1", 3001,
+                                   [{active, false}, binary]),
+    Req = <<"GET /", Path/binary, " HTTP/1.1\r\n">>,
+    gen_tcp:send(Socket, Req),
+    ?assertMatch({ok, <<"HTTP/1.1 414 Request-URI Too Long\r\n"
```

Comment:
Isn't the reason line just "URI Too Long" (without "Request" part)?

https://tools.ietf.org/html/rfc7231#section-6.5.12

Reply:
Same for 413 Payload Too Large